assert_test.go

```diff
@@ -0,0 +1,55 @@
+package gosnowflake
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+)
+
+func assertNil(t *testing.T, actual interface{}, descriptions ...string) {
+	if actual != nil {
+		desc := joinDescriptions(descriptions...)
+		errMsg := fmt.Sprintf("expected \"%s\" to be nil but was not%s", actual, desc)
+		t.Fatal(errMsg)
+	}
+}
+
+func assertNotNil(t *testing.T, actual interface{}, descriptions ...string) {
+	if actual == nil {
+		desc := joinDescriptions(descriptions...)
+		errMsg := fmt.Sprintf("expected to be not nil but was nil%s", desc)
+		t.Fatal(errMsg)
+	}
+}
+
+func assertEqual(t *testing.T, actual string, expected string, descriptions ...string) {
+	if expected != actual {
+		desc := joinDescriptions(descriptions...)
+		errMsg := fmt.Sprintf("expected \"%s\" to be equal to \"%s\" but was not%s", actual, expected, desc)
+		t.Fatal(errMsg)
+	}
+}
+
+func assertContains(t *testing.T, actual string, expectedToContain string, descriptions ...string) {
```

Comment:
Maybe it would be better to rename it to `stringContains`? There is no function overloading in Go.

Reply:
I renamed it to assertStringContains, because I think that it would be good to have assert in the method's name

Reply:
Yes, very good, it was only a brachylogy.